src/to_vec.jl

```diff
@@ -21,6 +21,13 @@ end
 # Base case -- if x is already a Vector{<:Real} there's no conversion necessary.
 to_vec(x::Vector{<:Real}) = (x, identity)
 
+# get around the outer constructors
+macro _force_construct(T, argsplat_ex)
+    @assert argsplat_ex.head == :(...)
```

Comment:
Maybe also we assert that it has only got 1 arg?
```suggestion
    @assert Meta.isexpr(argsplat_ex, :(...), 1)
```

Reply:
It can't have two, right? Would that not have errored earlier?

Reply:
idk.
Assertions are like excutable comments,
so also including that info seems useful